rules/prefer-string-replace-all.js

```diff
@@ -82,13 +84,37 @@ const create = context => ({
 			return;
 		}
 
+		const methodName = property.name;
+		const patternReplacement = getPatternReplacement(pattern);
+
+		if (methodName === 'replaceAll') {
+			if (!patternReplacement) {
+				return;
+			}
+
+			return {
+				node: pattern,
+				messageId: MESSAGE_ID_USE_STRING,
+				data: {
+					replacement: patternReplacement.length < 20 ? patternReplacement : 'literal',
```

Comment:
Can you add a code comment here? I don't really understand the purpose of the `< 20` check.